Dockerfile.build

```diff
@@ -1,7 +1,7 @@
 FROM gradle:5.4.1-jdk11
 LABEL maintainer="Rob March <rob.march@digirati.com>"
 
-RUN apt-get clean && apt-get update && apt-get -y upgrade
+RUN apt-get clean && apt-get update && apt-get -y upgrade \
```

Comment:
`Dockerfile` linting should be added to our pre-commit config.

Reply:
Hadolint is a good tool, here's a pre-commit config for it: https://github.com/stratasan/hadolint-pre-commit